pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -323,6 +322,92 @@ public enum Type {
       public static final String CONFIG_OF_MAX_RETRIES = "pinot.broker.failure.detector.max.retries";
       public static final int DEFAULT_MAX_RETIRES = 10;
     }
+
+    // Configs related to AdaptiveServerSelection.
+    public static class AdaptiveServerSelector {
+      /**
+       * Adaptive Server Selection feature has 2 parts:
+       * 1. Stats Collection
+       * 2. Routing Strategy
+       *
+       * Stats Collection is controlled by the config CONFIG_OF_ENABLE_STATS_COLLECTION.
+       * Routing Strategy is controlled by the config CONFIG_OF_TYPE.
+       *
+       *
+       * Stats Collection: Enabling/Disabling stats collection will dictate whether stats (like latency, #
+       *                   inflight requests) will be collected when queries are routed to/received from servers.
+       *                   It does not have any impact on the Server Selection Strategy used.
+       *
+       * Routing Strategy: Decides what strategy should be used to pick a server. The available strategies are as
+       *                   follows:
+       *                   1. NO_OP: Uses the default roundrobin approach. Does NOT require Stats Collection to be
```

Comment:
(nit) call it default behavior offered by existing instance selector strategies (balanced or RG)

Reply:
Done